lib/python/qmk/cli/new/keymap.py

```diff
@@ -0,0 +1,65 @@
+"""
+This script automates the copying of the default keymap into your own keymap.
+"""
+import os
+import sys
+
+from milc import cli
+from qmk.path import keymap, check_directories, create_keymap_directory
+
+
+# TODO(orngshdw): add debugging code if needed:
+#   @cli.argument('--debug', dest='debug', action='store_true', help='specify to turn on debugging messages')
+@cli.entrypoint('Create a new keymap.c from the default keyboard of your choosing')
+def main(cli):
+
+    # get keyboard
+    keyboard = input("Keyboard Name: ")
+    # get username
```

Comment:
It would be good to refine your comments a bit. This one and the previous one could probably collected into a single comment that says "Collect information the user didn't provide".